experiments/conductor/cmd/runner/runner.go

```diff
@@ -700,3 +701,40 @@ func splitMetadata(opts *RunnerOptions, branches Branches) {
 		}
 	}
 }
+
+func shouldKeepBranch(branch Branch) bool {
+	// Keep if Skip is false
+	if !branch.Skip {
+		return true
+	}
+
+	// Skip if gcloud command is empty
+	if branch.Command == "" {
+		return false
+	}
+
+	// Run gcloud xxx --help to see if it implements CRUD methods
+	gcloudCommand := branch.Command
```

Comment:
running this would front load a lot of gcloud commands before even any processing begins. I would add this as a -4 command which would set the skip field. 
Additionally i would add it to the notes section.